### ISSUE TITLE
Launch IPFS in docker compose

### ISSUE THREAD

**Issue by sea212** [zeitgeistpm/app-docker#3]
An IPFS node should be launched and made available for other docker containers within docker compose.

It regards to permanency of data, see https://github.com/zeitgeistpm/app-docker/issues/2#issuecomment-1433264231

**Comment by TvrtkoM** [comment]
Following commands need to be executed inside ipfs docker container to be able to use it by the app
```
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Methods '["GET", "POST"]
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Credentials '["true"]'
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\"*\"]"
 ```

and the ipfs container has to be restarted then for config to work.

other alternative is to use preconfigured config file and override the default one used by ipfs docker image.

it should have this section in the json

![image](https://user-images.githubusercontent.com/551965/220181287-4587c892-9199-4a3f-9a3e-ced42e3a086f.png)

**Comment by sea212** [comment]
Closed by #7
